.github/chainguard/self.update-kubernetes-versions.create-pr.sts.yaml

```
@@ -0,0 +1,13 @@
+---
+issuer: https://token.actions.githubusercontent.com
+
+subject: repo:DataDog/datadog-agent:environment:main
+
+claim_pattern:
+  event_name: (workflow_dispatch|schedule)
+  ref: (refs/heads/main|refs/heads/test_release/.*)
+  job_workflow_ref: DataDog/datadog-agent/\.github/workflows/create_rc_pr\.yml@refs/heads/(main|test_release/.*)
```

```suggestion
  job_workflow_ref: DataDog/datadog-agent/\.github/workflows/update-kubernetes-versions\.yml@refs/heads/main
```
You need to use the correct workflow name. And same as previously for the ref name